src/EnergyPlus/EcoRoofManager.cc

```diff
@@ -907,35 +909,47 @@ namespace EcoRoofManager {
         }
 
         // NEXT Add Precipitation to surface soil moisture variable (if a schedule exists)
-        if (!state.dataGlobal->WarmupFlag) {
-            state.dataEcoRoofMgr->CurrentPrecipitation = 0.0; // first initialize to zero
-        }
-        state.dataEcoRoofMgr->CurrentPrecipitation = 0.0; // first initialize to zero
-        if (state.dataWaterData->RainFall.ModeID == DataWater::RainfallMode::RainSchedDesign) {
-            state.dataEcoRoofMgr->CurrentPrecipitation = state.dataWaterData->RainFall.CurrentAmount; //  units of m
-            Moisture += state.dataEcoRoofMgr->CurrentPrecipitation / state.dataEcoRoofMgr->TopDepth;  // x (m) evenly put into top layer
-            if (!state.dataGlobal->WarmupFlag) {
-                state.dataEcoRoofMgr->CumPrecip += state.dataEcoRoofMgr->CurrentPrecipitation;
-            }
+        Moisture += state.dataEcoRoofMgr->CurrentPrecipitation / state.dataEcoRoofMgr->TopDepth; // x (m) evenly put into top layer
+
+        if (state.dataEnvrn->RunPeriodEnvironment) {
+            state.dataEcoRoofMgr->CumPrecip += state.dataEcoRoofMgr->CurrentPrecipitation;
```

Comment:
To match develop, this should be ` &&(!state.dataGlobal->WarmupFlag)`
Looking at the csv output, Environment:Green Roof Cumulative Precipitation Depth [m](Hourly) starts at zero in develop, and starts at over 5m in this branch.

Reply:
I changed it to `"state.dataGlobal->WarmupFlag"` now. I'll double-check the Environment:Green Roof Cumulative Precipitation Depth quantity